honeybee_schema/energy/hvac/allair.py

```diff
@@ -0,0 +1,180 @@
+    vav_dcw_pfp = 'VAV district chilled water with PFP boxes'
+    vav_dcw_gcr = 'VAV district chilled water with gas coil reheat'
+    vav_dcw_base = 'VAV district chilled water with no reheat with baseboard electric'
+    vav_dcw_guh = 'VAV district chilled water with no reheat with gas unit heaters'
+    vav_dcw_zhp = 'VAV district chilled water with no reheat with zone heat pump'
+
+
+class PVAVEquipmentType(str, Enum):
+    pvav_gbr = 'PVAV with gas boiler reheat'
+    pvav_ashp = 'PVAV with central air source heat pump reheat'
+    pvav_dhw = 'PVAV with district hot water reheat'
+    pvav_pfp = 'PVAV with PFP boxes'
+    pvav_ger = 'PVAV with gas heat with electric reheat'
+
+
+class PSZEquipmentType(str, Enum):
+    psz_e_base = 'PSZ-AC with baseboard electric'
+    psz_gb_base = 'PSZ-AC with baseboard gas boiler'
+    psz_dhw_base = 'PSZ-AC with baseboard district hot water'
+    psz_guh = 'PSZ-AC with gas unit heaters'
+    psz_ec = 'PSZ-AC with electric coil'
+    psz_gc = 'PSZ-AC with gas coil'
+    psz_gb = 'PSZ-AC with gas boiler'
+    psz_ashp = 'PSZ-AC with central air source heat pump'
+    psz_dhw = 'PSZ-AC with district hot water'
+    psz_ac = 'PSZ-AC with no heat'
+    psz_dcw_e_base = 'PSZ-AC district chilled water with baseboard electric'
+    psz_dcw_gb_base = 'PSZ-AC district chilled water with baseboard gas boiler'
+    psz_dcw_dhw_base = 'PSZ-AC district chilled water with baseboard district hot water'
+    psz_dcw_guh = 'PSZ-AC district chilled water with gas unit heaters'
+    psz_dcw_ec = 'PSZ-AC district chilled water with electric coil'
+    psz_dcw_gc = 'PSZ-AC district chilled water with gas coil'
+    psz_dcw_gb = 'PSZ-AC district chilled water with gas boiler'
+    psz_dcw_ashp = 'PSZ-AC district chilled water with central air source heat pump'
+    psz_dcw_dhw = 'PSZ-AC district chilled water with district hot water'
+    psz_dcw_ac = 'PSZ-AC district chilled water with no heat'
+    psz_hp = 'PSZ-HP'
+
+
+class PTACEquipmentType(str, Enum):
+    ptac_e_base = 'PTAC with baseboard electric'
+    ptac_gb_base = 'PTAC with baseboard gas boiler'
+    ptac_dhw_base = 'PTAC with baseboard district hot water'
+    ptac_guh = 'PTAC with gas unit heaters'
+    ptac_ec = 'PTAC with electric coil'
+    ptac_gc = 'PTAC with gas coil'
+    ptac_gb = 'PTAC with gas boiler'
+    ptac_ashp = 'PTAC with central air source heat pump'
+    ptac_dhw = 'PTAC with district hot water'
+    ptac = 'PTAC with no heat'
+    pthp = 'PTHP'
+
+
+class FurnaceEquipmentType(str, Enum):
+    furnace = 'Forced air furnace'
+
+
+class VAV(_AllAirBase):
+    """Variable Air Volume (VAV) HVAC system."""
+
+    type: constr(regex='^VAV$') = 'VAV'
+
+    equipment_type: VAVEquipmentType = Field(
+        VAVEquipmentType.vav_chill_gbr,
+        description='Text for the specific type of the system and equipment '
+        'as it appears in the OpenStudio standards gem.'
```

Comment:
I think we should mention the Enumeration and not OpenStudio standards gem. One should be able to use the schema without a reference to an OpenStudio gem. If we have to keep the reference then I would link it to a documentation as the gem itself can change over time.